Branches/Stable/Behaviors/obj_Hauler.iss

```diff
@@ -459,12 +459,24 @@ objectdef obj_Hauler
 		}
 		else
 		{
+			; TODO: find a way to check if FleetMembers.Peek is in station or not
 			if ${FleetMembers.Peek(exists)} && ${Local[${FleetMembers.Peek.Name}](exists)}
 			{
 				call This.WarpToFleetMemberAndLoot ${FleetMembers.Peek.CharID}
 			}
 			FleetMembers:Dequeue
 		}
+		
+		if !${FleetMembers.Peek(exists)}
+		{
+			call Safespots.WarpTo
+			wait 20
+			do
+			{
+				wait 5

```

Comment:
If you're in space, and trhe fleetmember is not, this wait loop will be infinite.
Add a check that it aborts after a few seconds, and switches to the next fleetmember, to correct tht